cognee/modules/data/processing/document_types/TextDocument.py

```diff
@@ -15,6 +15,8 @@ def get_text():
 
                     yield text
 
-        chunker = TextChunker(self, chunk_size = chunk_size, get_text = get_text)
+        chunker_func = ChunkerConfig.get_chunker(chunker)
```

Comment:
Why we don't pass the `chunker` function itself to `read` method?
That way we don't have to have this config and chunker can be customized.

Reply:
Do we want that map? It is an additional step to do in order to use a new chunker. What I see is that we need some chunker interface, so that all custom chunkers we add are compatible with the core.

Reply:
Lets discuss this in person